etna/search/templatetags/search_tags.py

```diff
@@ -61,32 +61,29 @@ def query_string_exclude(
     return query_dict.urlencode()
 
 
-@register.filter
-def include_hidden_fields(form_id_or_visible_field_names, form) -> str:
+@register.simple_tag
+def render_form_fields_as_hidden(form, exclude="", exclude_on_error="") -> str:
```

Comment:
What is `exclude_on_error` for? This tag is simply rendering hidden inputs. Any validation errors will be surfaced next to the editable versions of those fields. Remember why we're using this tag in the first place: we have multiple form elements  with different selections of visible/editable fields, yet when either form is posted, ALL values need to be preserved in the querystring, or they will be lost on the next request.